js/src/components/paid-ads/asset-group/asset-group-section.js

```diff
@@ -2,25 +2,27 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { useState, createInterpolateElement } from '@wordpress/element';
+import { createInterpolateElement } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
+import { useAdaptiveFormContext } from '.~/components/adaptive-form';
 import Section from '.~/wcdl/section';
 import AppDocumentationLink from '.~/components/app-documentation-link';
 import VerticalGapLayout from '.~/components/vertical-gap-layout';
 import FinalUrlCard from './final-url-card';
-import ImagesSelector from './images-selector';
-import TextsEditor from './texts-editor';
-import AssetField from './asset-field';
+import AssetGroupCard from './asset-group-card';
 import './asset-group-section.scss';
 
 /**
  * Renders the form content for managing an asset group of a campaign with Section UI.
+ *
+ * Please note that this component relies on an CampaignAssetsForm's context and custom adapter,
```

Comment:
Same as https://github.com/woocommerce/google-listings-and-ads/pull/1863/files#r1087449867

Seems like this is only used in `<AssetGroup>` so I guess we could remove the comment duplication.

Reply:
Same as https://github.com/woocommerce/google-listings-and-ads/pull/1863#discussion_r1090437704.